x-pack/plugins/searchprofiler/public/directives/_directives.scss

```diff
@@ -0,0 +1,148 @@
+.profilerDevTool {
+  
+  .panel {
+    background-color: inherit;
+    margin: 0;
+    padding: 0;
+    border: none;
+    border-bottom: 1px solid $euiColorDarkShade;
+  }
+
+  .shard-body {
+    padding-top: $euiSizeXS;
+    margin: 0;
+    margin-left: $euiSizeXS;
+  }
+
+  .panel-body-underline {
+    padding: $euiSize $euiSizeXS $euiSizeXS;
+    margin-top: $euiSize;
+    margin-left: $euiSizeS;
+  }
+
+  // Profile details treeview
+  .shard-details {
+    float: left;
+    margin-bottom: $euiSizeXS;
+
+    .fa {
+      width: $euiSizeM;
+      height: $euiSizeM;
+      cursor: pointer;
+    }
+  }
+
+  .shard-details-right {
+    color: $euiColorDarkShade;
+    float: right;
+    text-align: right;
+    margin-bottom: $euiSizeXS;
+  }
+
+  .shard {
+    margin-top: $euiSizeS;
+    min-width: 100%;
+    display:inline-block;
+    border: none;
+  }
+
+  .index {
+    margin-top: $euiSize;
+    min-width: 100%;
+    padding: $euiSizeS;
+    display: inline-block;
+  }
+
+  .tv-row:hover {
+    outline: solid 1px $euiColorMediumShade;
+  }
+
+  .tv-row-last-hover {
+    outline: solid 1px $euiColorMediumShade;
+  }
+
+  .tv-row,
+  .tv-row-header {
+    display: table;
+    width: 100%;
+    table-layout: fixed;
+  }
+
+  .tv-row-header {
+    margin-bottom: $euiSizeS;
+
+     > .cell {
+      color: $euiColorDarkShade;
+    }
+  }
+
+  .cell {
+    display: table-cell;
+    vertical-align: middle;
+    text-align: center;
+
+    > .detail {
+      font-style: italic;
+      font-size: $euiFontSizeXS;
+      padding-left: $euiSize;
+    }
+  }
+
+  .description {
+    background-color: $euiColorEmptyShade;
+    text-align: left;
+  }
+
+  .time, 
+  .totaltime {
+    width: 10%;
+  }
+  
+  .percentage {
+    width: 15%;
+    background-color: $euiColorEmptyShade;
+  }
+
+  .fa-bars {
+    padding-left: $euiSizeS;
+  }
+
+  // Progressbar
+  .graph {
+    height: 20px;
+    border: 1px solid $euiColorMediumShade;
+    background: linear-gradient(to top, $euiColorLightShade, $euiColorLightestShade 23%);
+    position: relative;
+    vertical-align: middle;
+    margin: $euiSizeXS;
+
+    p {
+      position: absolute;
+      text-align: center;
+      width: 100%;
+      margin: 0;
+      line-height: $euiFontSizeM;
+      vertical-align: middle;
+      font-size: $euiFontSizeXS;
+      order: 5;
+    }
+  }
+
+  .bar {
+    height: 20px; /* Not 30px because the 1px top-border brings it up to 30px to match #graph */
+    // SASSTODO This pink color is hardcoded elsewhere in the javascript and in tests, needs an overall redesign
```

Comment:
Ha. Man is this one ugly! Should we just use `$euiColorAccent`, with `color: chooseLightOrDarkText($euiColorAccent, #FFF, #000)` on it and call it a day?

At least then it'll work against theming

Reply:
I don't think that will quite get us there, unfortunately. 

The background fills like a progress bar, so it's possible for the text be on both light and dark backgrounds (e.g. around 50%; see below). Also, the two cells on the left are colored using logic in the app code and that color also appears to be part of a few tests. To further complicate things, the shades in the right hand panel also correlate to this baked in pink color.

<img width="805" alt="screenshot 2018-09-27 19 38 12" src="https://user-images.githubusercontent.com/446285/46181744-2adb6a00-c28d-11e8-8266-9bab423f08a3.png">

Given all that, I think it would be best to come back at this with a bit more involved redesign effort.